.travis.yml

```diff
@@ -1,9 +1,12 @@
-language: python
+language: minimal
+dist: xenial
 
-sudo: false
+env:
+  matrix:
+    - PYTHON_VERSION=3.7
 
-python:
-  - "2.7"
+git:
+  depth: 10000
```

Comment:
It might be worth adding a comment as to why this is so large, as [we have done in Iris](https://github.com/SciTools/iris/blob/master/.travis.yml#L31) ?

Reply:
@lbdreyer Done :smile: